src/foam/nanos/theme/Theme.js

```diff
@@ -928,14 +928,15 @@ foam.CLASS({
         var name = prop.name;
 
         if ( ! t1.hasOwnProperty(name) ) t1[name] = t2[name];
-        else if ( prop.compare(t1, t2) != 0 ) {
+        else if ( prop.compare(t1, t2) != 0 &&
+                  prop.compare(t2, foam.nanos.theme.Theme.create({}, this)) != 0 ) {
```

Comment:
Would it be better if you just called prop.isSet()? Same below.

Reply:
There is no prop.isSet() on the client, also the server already provides the default in the response body.